buildconfig/flatpak/Readme.md

```diff
@@ -1,15 +1,36 @@
-# Creating flatpak package manually
+# Creating Flatpak Package Manually
 
-Execute build.sh and install.sh (remove previous installations first)
+## Steps to Build and Install
 
-# Update to new tuxemon version
-- Update tag in flatpak manifest: org.tuxemon.Tuxemon.yaml
-- Update tag in updateRequirements.sh script
-- Update version of portmidi if exists
-- Execute script updateRequirements.sh to regenerate the requirements
-- Add build-options again to python3-pygame in python3-requirements.json
+1. **Remove Previous Installations**: Before proceeding, ensure that any previous installations of the package are removed.
 
-# Troubleshooting:
-## Pygame failed to build because portmidi is missing
-during execution of `updateRequirements.sh` the build config for pygame was removed from python3-requirements.json. Add it again:
-Build options for pygame copied from:https://github.com/flathub/com.katawa_shoujo.KatawaShoujo/blob/74e5f93c4a668789f6464ba13017a9737c12764b/pygame/pygame-1.9.6.json#L10-L41
\ No newline at end of file
+2. **Execute Build and Install Scripts**:
+   - Run `build.sh` to build the package.
+   - Run `install.sh` to install the package.
+
+## Update to New Tuxemon Version
```

Comment:
The manifest points to the dev branch, so the source is always up to date. In case of any build problems, because of changing in the requirements.txt, the manifest must be updated as described below.

Maybe: ## Update flatpak manifest

Reply:
@Murmele  updated